api/ordinals.ts

```diff
@@ -149,12 +149,14 @@ export async function getOrdinalsFtBalance(network: NetworkType, address: string
             balance: responseToken.overallBalance,
             total_sent: '0',
             total_received: '0',
-            principal: '',
+            principal: responseToken.ticker.toUpperCase(),
```

Comment:
why set the ticker as principal?

Reply:
`principal` represents a unique identifier for a given fungible token. For SIP-10 tokens, the `contract` is unique. For BRC-20, there is no contract, but the `ticker` is known to be unique.